module/zfs/vdev_trim.c

```diff
@@ -1305,8 +1307,9 @@ vdev_autotrim_thread(void *arg)
 			 * Allocate an empty range tree which is swapped in
 			 * for the existing ms_trim tree while it is processed.
 			 */
-			trim_tree = zfs_range_tree_create(NULL, ZFS_RANGE_SEG64,
-			    NULL, 0, 0);
+			trim_tree = zfs_range_tree_create_flags(
+			    NULL, ZFS_RANGE_SEG64, NULL, 0, 0,
+			    ZFS_RT_F_DYN_NAME, vdev_rt_name(vd, "trim_tree"));
```

Comment:
"autotrim_tree" would probably be helpful.

Reply:
Improved.